reo/nested_inputs.py

```diff
@@ -1833,6 +1907,307 @@ def list_of_dict(input):
+        },
+        "can_export_beyond_site_load": {
+          "type": "bool",
+          "default": False,
+          "description": "True/False for if technology can export energy beyond the annual site load (and be compensated for that energy at the wholesale_rate_above_site_load_us_dollars_per_kwh)."
+        },
+        "can_curtail": {
+          "type": "bool",
+          "default": False,
+          "description": "True/False for if technology can curtail energy produced."
+        },
+        "macrs_option_years": {
+          "type": "int",
+          "restrict_to": macrs_schedules,
+          "default": 0,
+          "description": "MACRS schedule for financial analysis. Set to zero to disable"
+        },
+        "macrs_bonus_pct": {
+          "type": "float",
+          "min": 0.0,
+          "max": 1.0,
+          "default": 0.0,
+          "description": "Percent of upfront project costs to depreciate under MACRS"
+        }        
+      },
+      "GHP": {
+        "force_ghp": {
+          "type": "bool",
+          "default": False,
+          "description": "Force one of the considered GHP design options"
+        },
+        "installed_cost_heatpump_us_dollars_per_ton": {
+          "type": "float",
+          "min": 0.0,
+          "max": 1.0e5,
+          "default": 1075.0,
+          "description": "Installed heating heat pump cost in $/ton (based on peak coincident cooling+heating thermal load)"
+        },
+        "heatpump_capacity_sizing_factor_on_peak_load": {
+          "type": "float",
+          "min": 1.0,
+          "max": 5.0,
+          "default": 1.1,
+          "description": "Factor on peak heating and cooling load served by GHP used for determining GHP installed capacity"
+        },
+        "installed_cost_ghx_us_dollars_per_ft": {
+          "type": "float",
+          "min": 0.0,
+          "max": 100.0,
+          "default": 14.0,
+          "description": "Installed cost of the ground heat exchanger (GHX) in $/ft of vertical piping"
+        },
+        "installed_cost_building_hydronic_loop_us_dollars_per_sqft": {
+          "type": "float",
+          "min": 0,
+          "max": 100.0,
+          "default": 1.70,
+          "description": "Installed cost of the building hydronic loop per floor space of the site"
+        },
+        "om_cost_us_dollars_per_sqft_year": {
+          "type": "float",
+          "min": -100,
+          "max": 100,
+          "default": -0.51,
+          "description": "Annual GHP incremental operations and maintenance costs in $/ft^2-building/year"
+        },
+        "building_sqft": {
+          "type": "float",
+          "min": 0.0,
+          "max": max_big_number,
+          "description": "Building square footage for GHP/HVAC cost calculations"
+        },
+        "ghpghx_inputs": {
+          "type": "list_of_dict",
+          "description": "GHP-GHX inputs/POST to /ghpghx endpoint"
+        },
+        "ghpghx_response": {
```

Comment:
is "ghpghx_response" necessary? Maybe just provide the id to get the response so that we aren't saving the same values in two places.

Reply:
I agree it is saving it in two places, but I'd like to keep it this way at least for now so that the reo/job runs with GHP can be re-created/copied without dependence on the state of a particular database.

Reply:
I changed this ghpghx_response to be ghpghx_response_uuids so we are just storing the ghp_uuids in the reo GHP inputs dict.